src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1877,7 +1865,7 @@ protected override void BeginProcessing()
             string message = string.Empty;
 
             // -Verb and -WindowStyle are not supported on non-Windows platforms as well as Windows headless SKUs
-            if (_isOnFullWinSku)
+            if (Platform.IsWindowsDesktop)
```

Comment:
Maybe cache the value in a variable instead of looking it up twice in the same method.